updatecli/values.yaml

```diff
@@ -0,0 +1,9 @@
+github:
+  epinio:
+    user: "GitHub Actions"
+    email: "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
what does this value match to?

Reply:
That's the "user" who commits using the github_token and github_actor
An example can be found [here](https://github.com/jenkins-infra/jenkins-infra/pull/2007)